lib_test/compat/user_program.expected

```diff
@@ -0,0 +1,43 @@
+
+# Parsed template
+
+## parsed
```

Comment:
I don't see this file being used anywhere in the test.

Reply:
`dune` compares this output with the output generated by running `user_program.exe` and fails if they differ.